Automattic/jetpack
# CLI: Add readme to 'jetpack release'

#### Changes proposed in this Pull Request:

Adds the `tools/plugin-changelog-to-readme.sh` to the CLI. 

#### Jetpack product discussion


#21160

#### Does this pull request change what data or activity we track or use?


No
#### Testing instructions:






* After running `jetpack release <project> changelog` and the CHANGELOG.md has been generated, follow the instructions to run `jetpack release <project> readme` 
* The readme.txt file should be updated properly.
* I don't believe we need to know if we are working with a beta or not, so you shouldn't be prompted for that unless you're using the changelog command.
* Any use cases that aren't covered?
